pulumi/infra/config.py

```diff
@@ -92,7 +101,7 @@ def _require_env_var(key: str) -> str:
 
 
 # Boy, this env name was not forward-thinking
-OPERATIONAL_ALARMS_EMAIL = _require_env_var("GRAPL_CDK_OPERATIONAL_ALARMS_EMAIL")
+OPERATIONAL_ALARMS_EMAIL = _require_env_var("GRAPL_OPERATIONAL_ALARMS_EMAIL")
```

Comment:
technically, this comment can be removed 🤣  but i won't force you to resubmit just for that

Reply:
🤦